src/test/javascript/spec/component/course/sidebar-item.service.spec.ts

```diff
@@ -0,0 +1,319 @@
+import { TestBed } from '@angular/core/testing';
+import { FeatureToggle } from 'app/shared/feature-toggle/feature-toggle.service';
+import {
+    faChalkboardUser,
+    faChartColumn,
+    faComments,
+    faFlag,
+    faGraduationCap,
+    faList,
+    faListAlt,
+    faPuzzlePiece,
+    faQuestion,
+    faRobot,
+    faTable,
+    faTableCells,
+    faUserCheck,
+} from '@fortawesome/free-solid-svg-icons';
+import { CourseSidebarItemService } from 'app/core/course/shared/sidebar-item.service';
+
+describe('CourseSidebarItemService', () => {
+    let service: CourseSidebarItemService;
+    const courseId = 123;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+        service = TestBed.inject(CourseSidebarItemService);
+    });
+
+    describe('getManagementDefaultItems', () => {
+        it('should return correct items with courseId', () => {
+            const items = service.getManagementDefaultItems(courseId);
+
+            expect(items.length).toBe(4);
```

Comment:
maybe it makes sense to check for equality on the whole array? This way, the test will immediately tell which elements is missing.